public/app/plugins/panel/canvas/ConnectionAnchors.tsx

```diff
@@ -11,6 +11,7 @@ type Props = {
 };
 
 export const CONNECTION_ANCHOR_DIV_ID = 'connectionControl';
+export const connectionAnchorAlt = 'connection anchor';
```

Comment:
Maybe we should update styling to be similar to other exported const i.e.
```suggestion
export const CONNECTION_ANCHOR_ALT = 'connection anchor';
```